src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -382,7 +402,7 @@ private static final class EnumMap {
      * }
      */
     @SuppressWarnings("removal")
-    private static MethodHandle generateInnerClass(MethodHandles.Lookup caller, Object[] labels) {
+    private static MethodHandle generateInnerClass(MethodHandles.Lookup caller, Class<?> selectorType, Object[] labels) {
```

Comment:
again a matter of style but it seems to me that the huge lambda inside of this method, starting in line 409, really wants to be a separate helper method. That will probably be a better refactoring as we won't be mixing byte code generation with method handles manipulation.

Reply:
Done